quick-start/src/main/webapp/WEB-INF/static/top/modal/flowModal.html

```diff
@@ -8,26 +8,41 @@ <h4 class="modal-title" id="myModalLabel">New Flow</h4>
       <div class="modal-body" id="myModalBody">
         <div class="row" ng-show="flowForm.hasErrors">
           <div id="flowModalMessage" class="modalMessage col-xs-12 col-md-12"></div>
-          </div>
+        </div>
         <div class="row">
-           <div id="details" class="col-xs-12 col-md-12">
-             <form ng-submit="saveFlow()">
-               <input name="domainName" type="hidden" ng-model="flowForm.domainName"/>
-               <input name="flowType" type="hidden" ng-model="flowForm.flowType"/>
-              <div class="form-group">
-                <label for="flowName">Flow Name</label>
-                <input class="form-control" name="flowName" type="text" ng-model="flowForm.flowName" placeHolder="Enter Flow Name" required="required"/>
-              </div>
-             </form>
-           </div>
+          <div id="details" class="col-xs-12 col-md-12">
+            <form ng-submit="saveFlow()">
+            <input name="domainName" type="hidden" ng-model="flowForm.domainName"/>
+             <input name="flowType" type="hidden" ng-model="flowForm.flowType"/>
+            <div class="form-group">
+              <label for="flowName">Flow Name</label>
+              <input class="form-control" name="flowName" type="text" ng-model="flowForm.flowName" placeHolder="Enter Flow Name" required="required"/>
+            </div>
+            <div class="form-group">
+              <label for="pluginFormat">Plugin Type</label>
+              <select name="pluginFormat" class="form-control" ng-model="flowForm.pluginFormat">
+                <option value="sjs">Javascript</option>
+                <option value="xqy">XQuery</option>
```

Comment:
There are multiple extensions that map to XQuery. Better to use mime-types here? Probably not if this is something that users would create manually.